mysite/base/tests.py

```diff
@@ -395,6 +397,7 @@ def get_dude():
         self.client.post(reverse(mysite.profile.views.unsubscribe_do), {'token_string': valid_token_string})
         self.assertFalse(get_dude().email_me_re_projects)
 
+    @expectedFailure
```

Comment:
Does this work the way I'd expect---i.e., makes this test a permitted failure? If so, that is a bit sad, but I am still happy to see the MySQL tests in the green and possible problematic tests clearly marked.